contracts/Core/BlockManager.sol

```diff
@@ -79,51 +70,25 @@ contract BlockManager is Initializable, ACL, BlockStorage {
         uint256[] memory medians,
         uint256 iteration,
         uint256 biggestInfluencerId
-    ) public initialized checkEpoch(epoch) checkState(parameters.propose()) {
+    ) external initialized checkEpoch(epoch) checkState(parameters.propose()) {
```

Comment:
Is this intended?

Reply:
yes. i dont know how it was set public. maybe erroneous conflict resolution in earlier prs